modules/op-about-taskrun.adoc

```diff
@@ -5,7 +5,7 @@
 [id="about-taskrun_{context}"]
 = TaskRun
 
-A _TaskRun_ instantiates a Task for execution with specific inputs, outputs, and execution parameters on a cluster. It can be invoked on its own or as part of a PipelineRun.
+A _TaskRun_ instantiates a Task for execution with specific inputs, outputs, and execution parameters on a cluster. It can be invoked on its own or as part of a PipelineRun for each Task in a pipeline.
```

Comment:
Please add this too: A TaskRun is automatically created by a PipelineRun for each Task in a Pipeline.

Reply:
Done!